Dangerfile

```diff
@@ -0,0 +1,59 @@
+
+has_source_changes = !git.added_files.grep(/Source.*\.swift/).empty?
+has_test_changes = !git.modified_files.grep(/Tests/).empty?
+
+###
+### Specific files to watch
+###
+
+# CI related files to watch for changes.
+@DANGER_CI_CD_FILES = ['.travis.yml', 'Dangerfile']
+
+# Dependency files to watch for changes.
+@DANGER_DEPENDENCY_FILES = ['Gemfile', 'JSQDataSourcesKit.podspec']
+
+# determine if any of the files were modified
+def didModify(files_array)
+	did_modify_files = false
+	files_array.each do |file_name|
+		if git.modified_files.include?(file_name) || git.deleted_files.include?(file_name)
+			did_modify_files = true
+		end
+	end
+	return did_modify_files
+end
+
+### WARNINGS
+
+warn('Changes to CI/CD files') if didModify(@DANGER_CI_CD_FILES)
+warn('Changes to dependency related files') if didModify(@DANGER_DEPENDENCY_FILES)
```

Comment:
Thinking about this more...

Actually, I don't think we need these specific warnings. These are rarely modified by contributors anyway, so I should always notice. If it ever becomes an issue, we can revisit.